tokio-util/src/io/sync_bridge.rs

```diff
@@ -6,6 +6,260 @@ use tokio::io::{
 
 /// Use a [`tokio::io::AsyncRead`] synchronously as a [`std::io::Read`] or
 /// a [`tokio::io::AsyncWrite`] as a [`std::io::Write`].
+///
+/// # Alternatives
+///
+/// In many cases, there are better alternatives to using `SyncIoBridge`, especially
+/// if you want to avoid blocking the async runtime. Consider the following scenarios:
+///
+/// When hashing data, using `SyncIoBridge` can lead to suboptimal performance and
+/// might not fully leverage the async capabilities of the system.
+///
+/// ### Why It Matters:
+///
+/// `SyncIoBridge` allows you to use synchronous I/O operations in an asynchronous
+/// context by blocking the current thread. However, this can be inefficient because:
+///  
```

Comment:
Now the spellcheck is failing because of trailing whitespace here and those other lines:
```
./tokio-util/src/io/sync_bridge.rs:22:///  
./tokio-util/src/io/sync_bridge.rs:25:///  
./tokio-util/src/io/sync_bridge.rs:29:///  
./tokio-util/src/io/sync_bridge.rs:237:///        
./tokio-util/src/io/sync_bridge.rs:240:///
```

Reply:
Done!